packages/angular/build/src/tools/esbuild/javascript-transformer-worker.ts

```diff
@@ -64,8 +65,15 @@ async function transformWithBabel(
   const { default: importAttributePlugin } = await import('@babel/plugin-syntax-import-attributes');
   const plugins: PluginItem[] = [importAttributePlugin];
 
-  // Lazy load the linker plugin only when linking is required
+  if (options.instrumentForCoverage) {
+    const { default: coveragePlugin } = await loadEsmModule<
+      typeof import('../babel/plugins/add-code-coverage.js')
+    >('../tools/babel/plugins/add-code-coverage.js');
```

Comment:
A direct dynamic import should work here.  Everything in the package is compiled to CJS currently so there shouldn't be any interop issues.

Reply:
Makes sense, thanks!